app/src/main/java/com/android/feedme/ui/camera/Camera.kt

```diff
@@ -157,16 +160,37 @@ fun CameraScreen(navigationActions: NavigationActions) {
                           imageVector = Icons.Default.PhotoCamera,
                           contentDescription = "Take photo")
                     }
-                // Icon for text recognition
+                // Button for text recognition
                 if (textRecognitionMode.value) {
                   IconButton(
                       onClick = { displayText.value = true },
-                      modifier = Modifier.testTag("MLButton")) {
+                      modifier =
+                          Modifier.size(56.dp)
+                              .background(CameraButtonsBackground, shape = CircleShape)
+                              .padding(10.dp)
+                              .testTag("MLTextButton")) {
                         Icon(
                             imageVector = Icons.TwoTone.TextFields,
                             contentDescription = "Display text after ML")
                       }
                 }
+                // Button for barcode scanner
+                if (barcodeRecognition.value) {
+                  val barcodeScannerPainter = painterResource(id = R.drawable.barcode_scanner)
+                  IconButton(
+                      onClick = { /*TODO scan barcode and display result*/},
+                      modifier =
+                          Modifier.size(56.dp)
+                              .background(CameraButtonsBackground, shape = CircleShape)
+                              .padding(10.dp)
+                              .testTag("MLBarcodeButton")) {
+                        Icon(
+                            painter = barcodeScannerPainter,
+                            contentDescription = "Barcode Scanner",
+                            modifier = Modifier.size(25.dp),
+                            tint = Color.Black)
```

Comment:
I would be better to use the Color theme of the App instead of `import androidx.compose.ui.graphics.Color`.

Reply:
Thanks you are right !